### PR TITLE
feat: add a README

### LINKED ISSUE #172: Update README
We need to look like pros 😎 
Let's do it before releasing to prod

### PR DESCRIPTION
**Pull request by nicosantangelo**
See here: https://github.com/decentraland/builder/blob/390c92f6725a5c1b961b7f254a915d819b278bfd/README.md

Closes #172

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,58 @@
...
+The builder is a SPA or single page application built with [create-react-app](https://github.com/facebook/create-react-app). It uses an [`.env`](#environment) file as configuration for a few things, you'll need to create that first.
+
+After that, to run this app you have two options:
+
+- _Development server_: run `npm start` from the [`root`](https://github.com/decentraland/builder/tree/master) path
+- _Production_: run `npm run build` and host the resulting index.html file with your server of choice, for example `python -m SimpleHTTPServer 5000`
+
+For more information, check the [create-react-app](https://github.com/facebook/create-react-app) repo.
+
+## Enviornment
```
Comment (cazala) [inline README.md]: Envi**ro**nment

**Comment by belohlavek** [comment]
Missing `REACT_APP_API_URL=` ?
Needs to be empty for the api wrapper

File: README.md (hunk 1 above)
Comment (nicosantangelo) [reply README.md]: <3

**Comment by belohlavek** [comment]
Should we talk about asset packs?

**Comment by nicosantangelo** [comment]
> Missing `REACT_APP_API_URL=` ?
> Needs to be empty for the api wrapper

It's not necessary to run the project so I didn't want to confuse anyone

**Comment by nicosantangelo** [comment]
> Should we talk about asset packs?

Didn't thought of that, do you have something in mind? I'll add a paragraph

**Pull request merged**